### PR TITLE
Add dependency on module puppetlabs/powershell

### PR DESCRIPTION
**Pull request by rigareau**
Fix for issue 23 Undocumented dependency on module puppetlabs/powershell

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -18,5 +18,9 @@
       "name": "puppetlabs/stdlib",
       "version_requirement": ">= 3.0.0 <5.0.0"
     }
```
Comment (electrical) [inline metadata.json]: You're missing a comma behind here.

**Comment by electrical** [comment]
Cool. Looks good to me. Failure is unrelated but known ( See IRC channel for details )

**Comment by igalic** [comment]
fix for #23

**Comment by alexjfisher** [comment]
Closed via https://github.com/voxpupuli/puppet-sslcertificate/issues/30
Thanks @rigareau !

**Pull request closed without merging**